### PR TITLE
Add mandatory chicken to bottom of readme

### PR DESCRIPTION
**Pull request by mitchhentges**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ Hello!
 
 This is a simple repository to demonstrate Git and Github usage for COMP4530, 
 Advanced Software Engineering at Thompson Rivers University.
+
+:chicken:
```
Comment (chrisfosterelli) [inline README.md]: I think this should be :cow2:

**Comment by chrisfosterelli** [comment]
Looks good, can we fix that emoji?

File: README.md (hunk 1 above)
Comment (mitchhentges) [inline README.md]: Not sure that I agree that it should be a :cow:, I feel that :chicken: better conveys the mandatory level of business synergy

Comment (chrisfosterelli) [inline README.md]: I don't think it should be :cow: either, it should be :cow2:

Comment (mitchhentges) [inline README.md]: Ah, missed that. Updated

**Comment by chrisfosterelli** [comment]
Thanks!

**Comment by mitchhentges** [comment]
np, thanks for the great project, has made my job easier every day

**Pull request merged**